src/modules/common/DateSelect.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import DatePicker from 'react-datepicker';
+
+import 'react-datepicker/dist/react-datepicker.css';
+
+class DateSelect extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      date: '',
+    };
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
You can remove the use of `bind` here and use an arrow function for handleChange (to give you access to `this`) instead.